src/Tribe/Importer/File_Importer.php

```diff
@@ -62,6 +63,11 @@ public static function get_importer( $type, Tribe__Events__Importer__File_Reader
 	public function __construct( Tribe__Events__Importer__File_Reader $file_reader, Tribe__Events__Importer__Featured_Image_Uploader $featured_image_uploader = null ) {
 		$this->reader = $file_reader;
 		$this->featured_image_uploader = $featured_image_uploader;
+
+		//Add Vendor Directory for Autoloading for Encoding Class
+		$autoloader = Tribe__Autoloader::instance();
+		$autoloader->register_prefix( 'Vendor__', Tribe__Events__Main::instance()->plugin_path . 'vendor' );
+		$autoloader->register_autoloader();
```

Comment:
I registered a vendor class here to use for the UTF8 Class...
Not sure if this is the best spot or it should be moved somewhere else. 
I made it generic so it could be used for other classes in the future.